.github/workflows/ci-cd.yml

```diff
@@ -63,7 +63,8 @@ jobs:
         run: pytest -v
 
       - name: Run docstring tests
-        if: github.event.pull_request.head.repo.fork != 'true'
+        # TODO(michael) disable doctests on 3.13 until workspace is updated to new Image Builder Version
```

Comment:
Preferably our doctests should be able to run on the mock servicer too... It's somewhere on my backlog 😬  I think it's currently blocked by pytest-asyncio fixtures not being usable